tools/slang-replay/main.cpp

```diff
@@ -79,14 +80,15 @@ int main(int argc, char *argv[])
     SlangRecord::RecordFileProcessor recordFileProcessor(options.recordFileName);
 
     Slang::String jsonPath = Slang::Path::replaceExt(options.recordFileName, "json");
-    SlangRecord::JsonConsumer jsonConsumer(jsonPath);
+    std::unique_ptr<SlangRecord::JsonConsumer> jsonConsumer;
```

Comment:
Just RefPtr<T>, no need to use std here.

Reply:
Done.